zeebe/engine/src/test/java/io/camunda/zeebe/engine/state/deployment/DecisionStateMultiTenantTest.java

```diff
@@ -105,6 +105,47 @@ void shouldPutDecisionForDifferentTenants() {
     assertDecision(latestDecisions2.get(0), tenant2, decisionKey, decisionId, version, drgKey);
   }
 
+  @Test
+  public void shouldStoreDecisionKeyByProcessIdAndDeploymentKeyForMultipleTenants() {
+    // given
+    final var drgKey = 123L;
+    final var drgId = "drgId";
+    final var decisionKey = 456L;
+    final var decisionId = "decisionId";
+    final int version = 1;
+    final var tenant1 = "tenant1";
+    final var tenant2 = "tenant2";
+    final var deploymentKey = 789L;
+    final var drg1 = sampleDecisionRequirementsRecord(tenant1, drgKey, drgId, version);
+    final var drg2 = sampleDecisionRequirementsRecord(tenant2, drgKey, drgId, version);
```

Comment:
💭 This test is a bit strange. It considers 2 DRGs belonging to different tenants, but they have the same key (`drgKey`). This already is the case in the test above (where this is likely copied from), so let's just accept it. It's not what this test focusses on, but just seems a bit strange.